plugins/main/public/components/common/modules/modules-defaults.tsx

```diff
@@ -44,7 +45,10 @@ import { mitreAttackColumns } from '../../overview/mitre/events/mitre-attack-col
 import { virustotalColumns } from '../../overview/virustotal/events/virustotal-columns';
 import { malwareDetectionColumns } from '../../overview/malware-detection/events/malware-detection-columns';
 import { WAZUH_VULNERABILITIES_PATTERN } from '../../../../common/constants';
-import { AlertsVulnerabilitiesDataSource } from '../data-source';
+import {
+  AlertsDataSource,
```

Comment:
Is it necessary to create the alertsThreatHuntingDataSource?

Reply:
Threat Hunting does not have a rule group filter, so the normal alert filter works